### DISCUSSION TITLE
Duda rango de años parte 2

### DISCUSSION THREAD

**Discussion by MaxAl100** [IIC2133-PUC/2024-1#66] [Tarea 1] [answered]
Tengo una duda respecto al rango de años que las cabezas tendrán; serán todas de 4 o menos dígitos o tendremos cabezas con años medidos en cientos de dígitos?
Desde ya muchas gracias :)

**Comment by benjavicente** [comment, marked as answer]
No asumas el tamaño en cantidad de dígitos. Eso sí, podrás asumir que está en el rango de 0 a 32767.
